UnityProject/Assets/Scripts/Systems/StationObjectives/Objectives/ShipResources.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using System.Text;
+using Strings;
+using Systems.Cargo;
+
+namespace StationObjectives
+{
+	/// <summary>
+	/// An objective to ship items to nanotrasen
+	/// </summary>
+	[CreateAssetMenu(menuName = "ScriptableObjects/StationObjectives/ShipResources")]
+	public class ShipResources : StationObjective
+	{
+		/// <summary>
+		/// The pool of possible resources to ship
+		/// </summary>
+		[SerializeField]
+		private ItemDictionary itemPool;
+
+		private List<Vector2> asteroidLocations = new List<Vector2>();
+		private ResourceTracker tracker;
+
+		protected override bool CheckCompletion()
+		{
+			if (tracker == null)
+			{
+				Logger.LogError($"Error, tracker is null!");
+				return Complete;
+			}
+			var finalReport = new StringBuilder();
+			finalReport.Append(victoryDescription);
+			finalReport.Replace("SHIPPEDVAL", $"{tracker.CurrentAmount}");
```

Comment:
I'd rather use ``Format`` instead of ``Replace``

Reply:
also why are we writting a  report in ``CheckCompletion`` ? By the name, I'd imagine the sole purpose of this method is to return true if the objective was completed. Consider adding fields for reports and methods to write them directly in the base Objective class.